app/react/Library/components/LibraryHeader.tsx

```diff
@@ -0,0 +1,138 @@
+import { Translate } from 'app/I18N';
+import React, { useEffect, useState } from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+import { wrapDispatch } from 'app/Multireducer';
+
+import { NeedAuthorization } from 'app/Auth';
+import { SortButtons } from 'app/Library/components/SortButtons';
+import LibraryModeToggleButtons from 'app/Library/components/LibraryModeToggleButtons';
+import {
+  zoomIn as zoomInAction,
+  zoomOut as zoomOutAction,
+} from 'app/Library/actions/libraryActions';
+import { IStore } from 'app/istore';
+import { IImmutable } from 'shared/types/Immutable';
+import { HiddenColumnsDropdown } from './HiddenColumnsDropdown';
+
+interface LibraryHeaderOwnProps {
+  storeKey: 'library' | 'uploads';
```

Comment:
Is it posible to remove this prop? The value should always be library.

Reply:
done